src/components/Auth.js

```diff
@@ -0,0 +1,91 @@
+import auth0 from 'auth0-js';
+
+let callbackURL;
+
+if(process.env.NODE_ENV === 'development'){
+    callbackURL = `http://localhost:3000/callback`
+} else if (process.NODE_ENV === 'production' || process.NODE_ENV !== 'development'){
+    callbackURL = `https://www.wellbroomed.com/callback`
+    // netlify callback URL goes here
+}
+
+class Auth {
+  constructor() {
+    this.auth0 = new auth0.WebAuth({
+      domain: process.env.REACT_APP_AUTH0_DOMAIN, // auth0 domain
+      audience: '', // intended JWT audience
+      clientID: process.env.REACT_APP_AUTH0_CLIENT_ID, // auth0 client id
+      redirectUri: `${callbackURL}`, // callback destination
+      responseType: 'id_token', // desired response (jwt)
+      scope: 'openid email profile' // user information requested
+    });
+
+    this.getProfile = this.getProfile.bind(this);
+    this.handleAuthentication = this.handleAuthentication.bind(this);
+    this.isAuthenticated = this.isAuthenticated.bind(this);
+    this.signIn = this.login.bind(this);
+    this.signOut = this.logout.bind(this);
```

Comment:
Could we have written the functions as an arrow function and the same name below so we wouldn't need these `.bind(this)` lines?

Reply:
Done, I think those were just carryovers from an older auth0 example, tested with arrow functions, seems to be working fine.